README.md

```diff
@@ -20,6 +20,7 @@ You can override the default values for the following variables:
 | Name                 | Description                                  | Default       |
 | -------------------- | -------------------------------------------- | ------------- |
 | `DBT_MZ_VERSION`     | `dbt-materialize` adapter version to install | `latest`      |
+| `POETRY_VERSION`     | Poetry version if dbt project is managed with poetry. Cannot be used with `DBT_MZ_VERSION` | |
```

Comment:
Wouldn't the Poetry version also be available in the user's `pyproject.toml`? Sounds a bit like a chicken and egg problem. I'm wondering if we could go from that file and `pip install .` instead of requiring users to set the Poetry version on the side. IIUC, this would not install dev dependencies by default (see [Poetry #3514](https://github.com/python-poetry/poetry/issues/3514)), but it seems like dependency management has changed quite a bit in [1.2.0](https://python-poetry.org/blog/announcing-poetry-1.2.0/#dependency-groups).